docs/graphcast/radios/one-shot.md

```diff
@@ -0,0 +1,35 @@
+---
+
+# 📟 One-shot CLI
+
+The source code for the one-shot CLI is available [on GitHub](https://github.com/graphops/subgraph-radio/tree/dev/one-shot) as a member of the Subgraph Radio workspace.
+
+## Introduction
+
+The one-shot CLI enables sending one-off messages. Currently, its default behaviour is to construct and send a message of type `VersionUpgradeMessage`, which is used for the [Subgraph Versioning functionality](graphcast/design-principles#subgraph-versioning) of Subgraph Radio.
+
+The one-shot CLI is configured using config variables. You will need to prepare the following config variables (either as env variables or passing CLI args when running the CLI):
+
+| Name                | Description and Examples                                                                                                                                                                                   |
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `PRIVATE_KEY`       | Private key to the Graphcast ID wallet (precendence over mnemonics).<br/>Example: `PRIVATE_KEY=YOUR_PRIVATE_KEY`                                                                                           |
+| `MNEMONIC`          | Mnemonic to the Graphcast ID wallet (first address of the wallet is used; Only one of private key or mnemonic is needed).<br/>Example: `MNEMONIC=YOUR_MNEMONIC`                                            |
+| `GRAPH_ACCOUNT`     | Graph account corresponding to Graphcast operator.<br/>Example: `GRAPH_ACCOUNT=YOUR_GRAPH_ACCOUNT`                                                                                                         |
+| `REGISTRY_SUBGRAPH` | Subgraph endpoint to the Graphcast Registry.<br/>Default: `https://api.thegraph.com/subgraphs/name/hopeyen/graphcast-registry-goerli`                                                                      |
+| `NETWORK_SUBGRAPH`  | Subgraph endpoint to The Graph network subgraph.<br/>Default: `https://api.thegraph.com/subgraphs/name/graphprotocol/graph-network-goerli`                                                                 |
+| `GRAPHCAST_NETWORK` | Supported Graphcast networks: mainnet, testnet.<br/>Default: `testnet`                                                                                                                                     |
+| `TOPICS`            | Comma separated static list of content topics to subscribe to.<br/>Example: `TOPICS=topic1,topic2,topic3`                                                                                                  |
+| `IDENTIFIER`        | Subgraph hash is used to be the message content identifier.<br/>Example: `IDENTIFIER=YOUR_IDENTIFIER`                                                                                                      |
```

Comment:
Maybe we should be a bit more specific: "Subgraph deployment hash", same with `NEW_HASH`